lib/node_modules/@stdlib/number/float64/base/exponent/src/exponent.c

```diff
@@ -0,0 +1,42 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/number/float64/base/exponent.h"
+#include "stdlib/number/float64/base/get_high_word.h"
+#include <stdint.h>
+
+/**
+* Extracts the exponent from a double-precision floating-point number.
```

Comment:
```suggestion
* Returns the unbiased exponent of a double-precision floating-point number.
```

To have consistency with other descriptions elsewhere in the package.